src/Core/src/Handlers/Label/LabelHandler.cs

```diff
@@ -46,7 +46,7 @@ public partial class LabelHandler : ILabelHandler
 		{
 		};
 
-		public LabelHandler() : base(Mapper)
+		public LabelHandler() : base(Mapper, CommandMapper)
```

Comment:
Don't forget to add docs for this :P 

I know that adding handler/mapper docs can (and probably should)  be automated since its basically "Handler for <Control>" or "Mapper for <Property>", but for the time being we have to do it manually

Reply:
Added all the docs!